ci-operator/templates/openshift/installer/cluster-launch-installer-src.yaml

```diff
@@ -0,0 +1,355 @@
+            cp "${KUBECONFIG}" /tmp/admin.kubeconfig
+            export KUBECONFIG=/tmp/admin.kubeconfig
+          fi
+          if ! oc get nodes 2>/dev/null; then
+            echo "Waiting for API at $(oc whoami --show-server) to respond ..."
+            sleep 15 & wait
+            continue
+          elif [[ -z "${API_UP}" ]]; then
+            echo "API at $(oc whoami --show-server) has responded"
+            API_UP=1
+          fi
+          if [[ -z "${ROUTER_NAMESPACE}" ]]; then
+            # check multiple namespaces while we are transitioning to the new locations
+            if oc get deploy/router -n tectonic-ingress 2>/dev/null; then
+              ROUTER_NAMESPACE=tectonic-ingress
+              ROUTER_DEPLOYMENT="deploy/router"
+            elif oc get ds/router-default -n openshift-ingress 2>/dev/null; then
+              ROUTER_NAMESPACE=openshift-ingress
+              ROUTER_DEPLOYMENT="ds/router-default"
+            elif oc get deploy/router -n openshift-ingress 2>/dev/null; then
+              ROUTER_NAMESPACE=openshift-ingress
+              ROUTER_DEPLOYMENT="deploy/router"
+            elif oc get deploy/router -n default 2>/dev/null; then
+              ROUTER_NAMESPACE=default
+              ROUTER_DEPLOYMENT="deploy/router"
+            else
+              echo "Waiting for router to be created ..."
+              sleep 15 & wait
+              continue
+            fi
+            echo "Found router in ${ROUTER_NAMESPACE}"
+          fi
+          break
+        done
+        i=0
+        MAX_RETRIES=10
+        wait_expiry_ts="$(($(date +%s) + 90))"
+        until oc --request-timeout=80s rollout status "${ROUTER_DEPLOYMENT}" -n "${ROUTER_NAMESPACE}" -w; do
+            i=$((i+1))
+            [ $i -eq $MAX_RETRIES ] && echo "timeout waiting for ${ROUTER_NAMESPACE}/${ROUTER_DEPLOYMENT} to be available" && exit 1
+            echo "error ${ROUTER_NAMESPACE}/${ROUTER_DEPLOYMENT} did not come up"
+            sleep "$((wait_expiry_ts - $(date +%s)))"
+            wait_expiry_ts="$(($(date +%s) + 90))"
+        done
+        # set up cloud-provider-specific env vars
+        if [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
+          export GOOGLE_APPLICATION_CREDENTIALS="/tmp/cluster/gce.json"
+          export KUBE_SSH_USER=cloud-user
+          mkdir -p ~/.ssh
+          cp /tmp/cluster/ssh-privatekey ~/.ssh/google_compute_engine || true
+          export PROVIDER_ARGS='-provider=gce -gce-zone=us-east1-c -gce-project=openshift-gce-devel-ci'
+        elif [[ "${CLUSTER_TYPE}" == "aws" ]]; then
+          export PROVIDER_ARGS="-provider=aws -gce-zone=us-east-1"
+          mkdir -p ~/.ssh
+          cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_aws_rsa || true
+        elif [[ "${CLUSTER_TYPE}" == "openstack" ]]; then
+          mkdir -p ~/.ssh
+          cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_openstack_rsa || true
+        fi
+        mkdir -p /tmp/output
+        cd /tmp/output
```

Comment:
```diff
-        mkdir -p /tmp/output
-        cd /tmp/output
```

We can't `cd` out of the repository's directory or `${TEST_COMMAND}` will likely fail.